src/config/runit/configuring.md

```diff
@@ -0,0 +1,44 @@
+# Configuring Services
+
+## Editing services
```

Comment:
I like this, but we should probably include a section before this one that mentions the conf files, since most services have options for necessary configuration included in their run files. In that case, the user can just modify that file and keep using the service directory provided by the package manager.

Reply:
Thanks for the feedback - PR updated.